x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/permission/IndicesPermission.java

```diff
@@ -246,6 +262,7 @@ public boolean check(String action) {
      * @param checkForIndexPatterns check permission grants for the set of index patterns
      * @param allowRestrictedIndices if {@code true} then checks permission grants even for restricted indices by index matching
      * @param checkForPrivileges check permission grants for the set of index privileges
+     * @param combineIndexGroups combine index groups to enable checking against regular expressions
```

Comment:
If we want to use this in `hasPrivileges` this boolean can be removed.

Reply:
Let's keep it as it is for now and decide if we want to consolidate it in a follow up.